src/main/java/core/basesyntax/Figures.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public abstract class Figures implements Drawable, Displayable {
+    private Color color;
+
+    public Figures(Color color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color.toString();
+    }
+
+    public abstract double getArea();
+
+    @Override
+    public void draw() {
```

Comment:
remove this method, we are not obligated to override interfaces method in the abstract class if we gonna do it in child classes